nessie/lib/queries.py

```diff
@@ -40,6 +40,10 @@ def advisor_schema():
     return app.config['REDSHIFT_SCHEMA_ADVISOR']
 
 
+def advisor_schema_internal():
+    return app.config['REDSHIFT_SCHEMA_ADVISOR_INTERNAL']
```

Comment:
@paulkerschen - Should `config/default.py` have `REDSHIFT_SCHEMA_ADVISOR_INTERNAL`?

Reply:
@johncrossman Yes, it already does - we just weren't querying it Nessie-side till now.